src/django_components/extensions/dependencies.py

```diff
@@ -16,6 +16,14 @@ class DependenciesExtension(ComponentExtension):
 
     # Cache the component's JS and CSS scripts when the class is created, so that
     # components' JS/CSS files are accessible even before having to render the component first.
+    #
+    # This is important for the scenario when the web server may restart in a middle of user
```

Comment:
Also not related to the main feature. I just noticed that I forgot why this code was here, so documented it.